.github/config/nodejs-prod.jsonc

```diff
@@ -76,8 +76,6 @@
 
     // TODO: fix these
     "ai-platform/snippets", // PERMISSION_DENIED: Permission denied: Consumer 'projects/undefined' has been suspended.
-    "appengine/analytics", // [ERR_REQUIRE_ESM]: require() of ES Module
-    "appengine/metadata/flexible", // [ERR_REQUIRE_ESM]: require() of ES Module
```

Comment:
This isn't documented anywhere, but we've been trying to keep the -prod and -dev config files in-sync, such that tests are only run in one or the other. Could I ask you to add this exclusion to the -dev config now that these are expected to work in the prod config?

Reply:
There might be a few other PRs that didn't do this, I'll clean this up here and create a new PR to fix.